cloud/shared/bin/lib/setup.py

```diff
@@ -17,106 +17,107 @@
 The script generates a .tfvars file that is used to deploy via terraform.
 """
 
-###############################################################################
```

Comment:
High level comment for next time, nothing to do here.

If possible, can we do things like this as independent PRs? The diff for moving to a `main()`-based approach is the majority of this PR. Finding the place where auto-approve was removed required looking at a fair number of removed lines on the left and ensuring that it wasn't on the right side of the diff.

Reply:
Yes, I should have done it like that in the first place. Moved "main-ifying" to separate PR: https://github.com/civiform/civiform/pull/3041